src/EnergyPlus/CurveManager.hh

```diff
@@ -82,7 +82,7 @@ namespace CurveManager {
 
     enum class CurveTypeEnum
     {
-        Unassigned,
+        Unassigned = -1,
         Linear,
```

Comment:
Same comment as above. An `enum` is a type. Calling something `XTypeEnum` is the same as calling it `XTypeType`.